android/app/src/main/java/org/uwaterloo/subletr/di/SingletonModule.kt

```diff
@@ -24,6 +24,8 @@ import javax.inject.Singleton
 @Module
 @InstallIn(SingletonComponent::class)
 object SingletonModule {
+	private const val apiBaseUrl = "https://subletr.duckdns.org/api"
```

Comment:
@andr-dev is here good or should this be in `ApiClient.kt`?

Reply:
Nope, this is perfect!
We should never modify any of the files in the api folder since they're autogenerated by OpenAPI. Any changes would be wiped the next time I run the OpenAPI build command.

Reply:
oh yeah got it

A haiku for your troubles
How many times will
I have to be reminded
I will remember